docs_assistant/contributors.py

```
@@ -414,4 +557,66 @@ def update_special_thanks_file():
+            logger.error("Failed to fetch contributors and sponsors data")
+            return False
+        
+        # 获取当前时间
+        current_time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        
+        # 格式化基础内容
+        base_content = f"""# 🙏 Special Thanks\n\n
+
+The development of New API would not be possible without the support and contributions of the community. We would like to express our special gratitude to all individuals and organizations who have helped with this project.
+
+"""
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)